src/test/regress/expected/binary_protocol.out

```diff
@@ -25,6 +25,43 @@ SELECT * FROM t ORDER BY id;
  10
 (10 rows)
 
+EXPLAIN (ANALYZE TRUE, TIMING FALSE, COSTS FALSE, SUMMARY FALSE) SELECT id FROM t ORDER BY 1;
+                            QUERY PLAN
+---------------------------------------------------------------------
+ Sort (actual rows=10 loops=1)
+   Sort Key: remote_scan.id
+   Sort Method: quicksort  Memory: 25kB
+   ->  Custom Scan (Citus Adaptive) (actual rows=10 loops=1)
+         Task Count: 2
+         Data received: 11 bytes
+         Tasks Shown: One of 2
+         ->  Task
+               Data received: 8 bytes
```

Comment:
I would change the wording here to make clear this is what the worker executing this task send to the coordinator. Now it might be interpreted as if the worker received this amount of data from the coordinator instead.

Reply:
I changed the wording to "Data received from worker" for the `Task` and "Data received from workers" for the `Custom Scan`. I agree that that's clearer.